sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOperation.cs

```diff
@@ -82,6 +62,7 @@ public RecognizeContentOperation(string operationId, FormRecognizerClient client
             Id = operationId;
             _serviceClient = client.ServiceClient;
             _diagnostics = client.Diagnostics;
```

Comment:
Pass client.Diagnostics directly and remove the field? Or is it still needed?

Reply:
It's still being used in `GetOperationFailedException` to create the `RequestFailedException`.